package.json

```diff
@@ -1,17 +1,15 @@
 {
   "name": "magicat",
-  "version": "1.5.5",
-  "description": "",
+  "version": "1.6.0",
+  "description": "Deep learning magic.. with the convenience of cat!",
   "files": [
-    "app.js",
-    "model"
+    "app.js"
   ],
   "bin": {
     "magicat": "app.js"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^0.13.4",
-    "@tensorflow/tfjs-node": "^0.1.20",
+    "@codait/max-image-segmenter": "^0.1.4",
```

Comment:
We'll need to update this to `v0.1.7` I think. I'm still getting the version with logs.

Reply:
done